docs/apis-tools/camunda-api-rest/camunda-api-rest-data-fetching.md

```diff
@@ -0,0 +1,324 @@
+---
+id: camunda-api-rest-data-fetching
+title: "Data fetching"
+description: "Learn about fetching data using the Orchestration Cluster API."
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+All persistent resources like process definitions, user tasks, users, or tenants provide at least one endpoint to fetch related data. Most of those endpoints provide data with **near-real time consistency** queried from exported records, if records for the respective resource are exported by the engine. If resources are not based on exported records, such as license data or topology information, the data returned by those endpoints can reflect real time insights or static content. For resources like messages, signals, or the engine's internal clock, the engine does not persist any specific data and thus there is no Orchestration Cluster API to fetch them.
+
+Many resources offer search endpoints to query by POST method and a given query request object where applicable. The structure of such search requests always follows the same schema and so does the response, always returning a list of items matching the query criteria, accompanied by search result metadata.
+
+[Search requests](#search-requests) are forwarded as queries to the data stores that hold the exported records and the search results are returned in the format described in [search responses](#search-responses).
```

Comment:
💭 could be shortened or omitted based on same thoughts as previous comment

Reply:
Yes, I agree 👍